### PR TITLE
Gaia project

### PR DESCRIPTION
**Pull request by zhiyanwang**
Here are the results from my part of the Gaia project. I successfully created a transverse velocity distribution for Gaia stars with high signal to noise ratio (velocity_distribution_2D.png). There are also two plots showing the density distribution of stars and the histogram of all velocity's magnitudes.

The only code you need to review is "complete_code_velocity_distribution_2D.py".

Pytest passes on my local computer, but on GitHub it requires the data file "tgas-source.fits" that is too large to be uploaded (693MB). This is why the Travis test is failing.

### CODE REVIEW

**Comment by kelle** [comment]
@drewjrosen Please provide Zhiyan some feedback on this PR.

File: .travis.yml
Comment (drewjrosen) [inline .travis.yml]: Add python 2 test.

**Review by drewjrosen — COMMENTED** [review]
Maybe somewhere on your plots, it should include the SN cutoff?

File: .travis.yml
Comment (zhiyanwang) [reply .travis.yml]: I'm not sure what you mean. What I plotted is already the data with signal to noise ratio >= 16, as explained repeatedly in code's comments, in this pull request's comment, and in the README file. I also included a density plot with all data and no SN cutoff  just for comparison.

Also, I'm curious why does it need to use python 2 test while 3.5 is working?

**Comment by zhiyanwang** [comment]
@kelle I haven't heard back from Drew since I replied him 6 days ago. Am I waiting for him to approve the pull request? Or is there anything else I need to do to fulfill all the course requirements? Please let me know. Thank you.

**Comment by drewjrosen** [comment]
Sorry I've been busy doing qual prep.

**Comment by drewjrosen** [comment]
It looks like I'm not allowed to merge your branch into the master.

**Review by kelle — APPROVED** [review]
Based on Drew's comments, I'm approving this PR. @zhiyanwang, after you merge this PR, this project will be done. :)

**Comment by zhiyanwang** [comment]
@kelle @drewjrosen Thank you very much for the semester!

**Pull request merged**
